#!/usr/bin/env python
PACKAGE = "robot_flock"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("numRobots",    int_t,    0, "number of robots", 6,  0, 100)
gen.add("radius",    double_t,    0, "radius of neigberhood", 2.0,  0.0, 10.0)
gen.add("separationStrength",    double_t,    0, "stregth of separation behavior", 0.1,  0.0, 10.0)
gen.add("cohesionStrength"  ,    double_t,    0, "stregth of cohesion behavior", 0.0,  0.0, 10.0)
gen.add("alignmentStrength" ,    double_t,    0, "stregth of alignment behavior", 0.0,  0.0, 10.0)
gen.add("formationStepSize" ,    double_t,    0, "step size of formation consensus", 0.5,  0.0, 1.0)



exit(gen.generate(PACKAGE, "robot_flock", "robotFlock"))